### DISCUSSION TITLE
Conversion from Rollup Helper?

### DISCUSSION THREAD

**Discussion by mrtrash757** [jamessimone/apex-rollup#264] [Ideas]
I see there is some code to automatically convert DLRS rollups to Apex Rollup, but was wondering if there was the same for Rollup Helper?

**Comment by jamessimone** [comment]
@mrtrash757 good to see you here! There is not currently a script for rollup helper’s metadata to apex rollup, but it sounds like it could be a great community contribution! I have had it in the back of my head for a while but haven’t had the chance to do something like that myself

**Comment by mrtrash757** [comment]
No problem! Thank you for the answer.

On Tue, Mar 8, 2022 at 12:28 PM James Simone ***@***.***>
wrote:

> @mrtrash757 <https://github.com/mrtrash757> good to see you here! There
> is not currently a script for rollup helper’s metadata to apex rollup, but
> it sounds like it could be a great community contribution! I have had it in
> the back of my head for a while but haven’t had the chance to do something
> like that myself
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jamessimone/apex-rollup/discussions/264#discussioncomment-2318346>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYECTJR3LPKD4WAHMSUZ7RDU66E4TANCNFSM5QGTORZA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Mahalo!

*Ross *🦝 🛩️ 🍕

Click here to book a meeting with me.  <http://calendly.com/rosskinkade>
